### PR TITLE
Add annotations to AR config for static upstreams IAM, Exhibitor, Diagnostics, Log, Net, Pkgpanda, Metrics, Checks, Mesos DNS

### PR DESCRIPTION
**Pull request by ahairape**
## High-level description
This PR is continuation of the work started earlier: https://github.com/dcos/dcos/pull/3996
It annotates the following upstreams in AR configuration files: Exhibitor, DC/OS Net, DC/OS Log, DC/OS Diagnostics.

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:
https://jira.mesosphere.com/browse/DCOS-47256

## Checklist for all PRs

  - [ ] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

### CODE REVIEW

**Comment by ahairape** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by ahairape** [comment]
@timaa2k @jongiddy @mhrabovcin PTAL, thank you

File: packages/adminrouter/extra/src/test-harness/tests/test_metrics.py [hunk 1]
```diff
@@ -35,6 +34,46 @@ def test_metrics_prometheus(self, master_ar_process):
...
+        static_upstream_annotations = {
+            '/acs/api/v1': 'IAM',
+            '/system/health/v1': 'DCOSDiagnostics',
+            '/system/checks/v1': 'DCOSChecks',
+            '/navstar/lashup/key': 'DCOSNet',
+            '/net/': 'DCOSNet',
+            '/pkgpanda': 'Pkgpanda',
+            '/system/v1/logs/': 'Log',
+            '/system/v1/metrics/': 'DCOSMetrics',
+            'exhibitor': 'Exhibitor',
```
Comment (timaa2k) [inline packages/adminrouter/extra/src/test-harness/tests/test_metrics.py]: Exhibitor is missing a slash here I suppose.

Comment (timaa2k) [reply packages/adminrouter/extra/src/test-harness/tests/test_metrics.py]: It should probably make a request to `/exhibitor/exhibitor/v1/cluster/status` instead.

File: packages/adminrouter/extra/src/test-harness/tests/test_metrics.py [hunk 2]
```diff
@@ -35,6 +36,55 @@ def test_metrics_prometheus(self, master_ar_process):
...
+        annotated for static upstreams
+        """
+        # Adding upstreams present in both Open and EE DC/OS
+        static_upstream_annotations = {
+            '/acs/api/v1': 'IAM',
+            '/system/health/v1': 'DCOSDiagnostics',
+            '/system/checks/v1': 'DCOSChecks',
+            '/navstar/lashup/key': 'DCOSNet',
+            '/net/': 'DCOSNet',
+            '/system/v1/logs/': 'Log',
```
Comment (timaa2k) [inline packages/adminrouter/extra/src/test-harness/tests/test_metrics.py]: This should check for `DCOSLog`

File: packages/adminrouter/extra/src/test-harness/tests/test_metrics.py [hunk 3]
```diff
@@ -35,6 +36,55 @@ def test_metrics_prometheus(self, master_ar_process):
         assert resp.headers['Content-Type'] == 'text/plain'
         assert resp.text.startswith('# HELP nginx_vts_info Nginx info')
 
+    def test_metrics_prometheus_static_upstreams_annotated(self, master_ar_process):
+        """
+        /nginx/metrics returns metrics in Prometheus format that are properly
+        annotated for static upstreams
+        """
+        # Adding upstreams present in both Open and EE DC/OS
+        static_upstream_annotations = {
```
Comment (jongiddy) [inline packages/adminrouter/extra/src/test-harness/tests/test_metrics.py]: Can we use `pytest.mark.parametrize` here so each of these checks runs as a separate test?

Comment (adamtheturtle) [reply packages/adminrouter/extra/src/test-harness/tests/test_metrics.py]: That also means we could have a separate test for the `repo_is_ee` case which lives in Enterprise - that seems better and more conventional!

**Comment by jongiddy** [comment]
Now that #4868 has merged, you might want to rebase against `master` to ensure the tests don't conflict.

File: packages/adminrouter/extra/src/test-harness/tests/test_metrics.py (hunk 3 above)
Comment (ahairape) [reply packages/adminrouter/extra/src/test-harness/tests/test_metrics.py]: @jongiddy I have parametrized the test
@adamtheturtle with the test parametrized it would be exactly the same code/function for EE and Open with only difference being the parametrization list, so it seems to me as a code duplication. (that's why I haven't implemented it)

Comment (timaa2k) [reply packages/adminrouter/extra/src/test-harness/tests/test_metrics.py]: I'd also like to see a test for Enterprise, even if it means to duplicate stuff initially. One day they will diverge (as almost anything in DC/OS EE).

Comment (ahairape) [reply packages/adminrouter/extra/src/test-harness/tests/test_metrics.py]: @timaa2k I have added Open- and EE-specific versions of the static endpoint annotation test, PTAL.
